src/domains/transaction/pages/SendRegistration/SendRegistration.tsx

```diff
@@ -45,7 +46,12 @@ export const SendRegistration = () => {
 
 	const { env } = useEnvironmentContext();
 	const activeProfile = useActiveProfile();
-	const activeWallet = useActiveWallet();
+	const { walletId } = useParams<{ walletId?: string }>();
+
+	const urlWallet = useActiveWalletWhenNeeded(walletId !== undefined);
+
+	const [activeWallet, setActiveWallet] = useState<Contracts.IReadWriteWallet | undefined>(urlWallet);
```

Comment:
@goga-m Because when we open the registration form (specifically the username registration form) from the deeplink, we don't know the wallet yet; we need to choose it from the select field.

Something to notice is that the rest of the registration forms do not have deeplink yet but may have in the future. That's the reason I'm asserting the wallet with assertWallet, so it's not handled as nullable there.